packages/core/src/App/Containers/RealAccountSignup/account-wizard.jsx

```diff
@@ -11,9 +11,11 @@ import AcceptRiskForm from './accept-risk-form.jsx';
 import LoadingModal from './real-account-signup-loader.jsx';
 import { getItems } from './account-wizard-form';
 import { useIsClientHighRiskForMT5 } from '@deriv/hooks';
-import 'Sass/details-form.scss';
 import { Analytics } from '@deriv-com/analytics';
 
+import 'Sass/details-form.scss';
+import 'Sass/account-wizard.scss';
+
```

Comment:
@ahmadtaimoor-deriv account-wizard is only used for signup flow. We have another modals in real account signup for trading assessement flow, set currency , add or mange modls etc. Moving the import to account -wizard is breaking styles for other modals

Reply:
It is reverted back to how it was . The broken scss after lazy-loading this modal is now handled by adding scss in wizard.scss . Thanks.